docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -2773,27 +3133,32 @@ The default server configuration file `config.xml` contains the following settin
 </crash_log>
 ```
 
-## backup_log
+## custom_cached_disks_base_directory
 
-Settings for the [backup_log](../../operations/system-tables/backup_log.md) system table for logging `BACKUP` and `RESTORE` operations.
+This setting specifies the cache path for custom (created from SQL) cached disks.
```

Comment:
L. 3138: Not my area of expertise. Since your new docs don't seem copied from config.xml, ... where are they exactly from?

Reply:
I've used the change log descriptions where available